algebras/algebra/src/main/scala/endpoints/algebra/Endpoints.scala

```diff
@@ -52,16 +52,18 @@ trait EndpointsWithCustomErrors extends Requests with Responses with Errors {
   ): Endpoint[A, B]
 
   /**
-    * @param summary     Short description
-    * @param description Detailed description
-    * @param tags        OpenAPI tags
-    * @param callbacks   Callbacks indexed by event name
+    * @param summary      Short description
+    * @param description  Detailed description
+    * @param tags         OpenAPI tags
+    * @param callbacks    Callbacks indexed by event name
+    * @param isDeprecated Declares this endpoint to be deprecated.
     */
   case class EndpointDocs(
     summary: Documentation = None,
     description: Documentation = None,
     tags: List[String] = Nil,
-    callbacks: Map[String, CallbacksDocs] = Map.empty
+    callbacks: Map[String, CallbacksDocs] = Map.empty,
+    isDeprecated: Boolean = false
```

Comment:
To stay consistent with what we have in `Operation` shouldn't we have simply `deprecated`? If don't find the `is` necessary.

Reply:
I thought it would be more “scala-ish”, but I agree that it’s probably simpler to use the OpenAPI vocabulary everywhere.